apollo-router/Cargo.toml

```diff
@@ -265,10 +267,10 @@ tikv-jemallocator = "0.5"
 
 [dev-dependencies]
 axum = { version = "0.6.20", features = [
```

Comment:
Not this PR, but I keep wondering why we duplicated entries in `dependencies` into `dev-dependencies`. We shouldn't unless the features set is different.